frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -526,7 +529,13 @@ function DataFrame({
     tooltip,
     clearTooltip,
     onItemHovered: handleTooltips,
-  } = useTooltips(columns, getCellContent)
+  } = useTooltips(
+    columns,
+    getCellContent,
+    // If dynamic editing is enabled, we need to ignore the last row (trailing row)
+    // because it would result in some undesired errors in the tooltips.
+    isDynamicAndEditable ? [numRows] : []
```

Comment:
does the indexing starts at 1? otherwise, `numRows` would be the length of the rows but not the index of the last row, right?

Reply:
indexing starts at 0, so just using numRows would point to the trailing row (-> which is numRows+1). I will add a comment to make that clear

Reply:
Added a comment about this.